src/Symfony/Component/Messenger/Tests/Fixtures/CallbackReceiver.php

```diff
@@ -22,4 +30,41 @@ public function receive(callable $handler): void
     public function stop(): void
     {
     }
+
+    public function getResponder(): ?ReceiverResponderInterface
+    {
+        return $this;
+    }
+
+    public function acknowledgeCurrentMessage(): void
```

Comment:
Could we replace `...CurrentMessage()` by `last...Message()`? The second pattern is more commonly used in Symfony, such as `getLastAuthenticationError()` and `getLastUsername()` in AuthUtils.

Reply:
the current is now gone from this for other reasons, fortunately :)